src/main.rs

```diff
@@ -66,6 +66,14 @@ impl Args {
             command.arg("-NoBattlEye");
         }
 
+        // if this was reversed in order, it would overwrite MaxPlayers in the GUS
+        // when not specified it is set to 70
+        if let Some(max_players) = game_user_settings.max_players() {
```

Comment:
The CLI arguments should override the config arguments, not the other way around.